tapir/translations/locale/de/LC_MESSAGES/django.po

```diff
@@ -142,7 +142,39 @@ msgid "Save"
 msgstr "Speichern"
 
 #: accounts/templates/accounts/email/create_account_reminder.body.html:6
-#, python-format
+#, fuzzy, python-format
+#| msgid ""
+#| "\n"
+#| "        <p>Hi %(display_name_short)s,</p>\n"
+#| "        <p>We've noticed that you've been a SuperCoop member for a month/week but haven't set up a Tapir account to sign up for your first shift.</p>\n"
+#| "        <p>\n"
+#| "            As a reminder, to sign up for your first shift (or indicate that you are eligible for an\n"
+#| "            <a href=\"https://docs.google.com/document/d/19v91mj2MAJTtrPLCw54VMS6qlaiJqBqUAuiC6z_AKpw/edit#heading=h.rjj2v2m92fqa\">exemption</a>),\n"
+#| "            please visit the Member Office in our shop.\n"
+#| "            There, an account will be set up for you on Tapir, our online shift management system, \n"
+#| "            and you will be guided through the steps to select your shift and working group.\n"
+#| "        </p>\n"
+#| "        <p>\n"
+#| "            For general questions about your membership, please contact the Member Office by email <a href=\"mailto:mitglied@supercoop.de\">(mitglied@supercoop.de)</a>\n"
+#| "            or visit <a href=\"https://www.google.com/maps/place/SuperCoop+Berlin/@52.553371,13.3585897,15z/data=!4m5!3m4!1s0x0:0x72e80db955fa8d3!8m2!3d52.5533091!4d13.3586012\">SuperCoop</a>.\n"
+#| "            Feel free to drop by and meet other SuperCoopies in person! Opening hours are::\n"
+#| "        <ul>\n"
+#| "            <li>Monday - Friday from 4:30 to 7:30 p.m.</li>\n"
+#| "            <li>Saturdays from 11 a.m. to 2 p.m.</li>\n"
+#| "        </ul>\n"
+#| "        </p>\n"
+#| "        <p>\n"
+#| "            How do you find SuperCoop?\n"
+#| "        </p>\n"
+#| "        <p>\n"
+#| "            You can find our shop at Oudenarder Straße 16, 13347 Berlin, on the corner of Seestraße. \n"
+#| "            There is also a back entrance to the shop, which you can reach via the former Osramhöfe. This entrance is also barrier-free.\n"
+#| "        </p>\n"
+#| "        <p>\n"
+#| "            Cooperative Greetings,<br />\n"
+#| "            The Member Office\n"
+#| "        </p>\n"
+#| "    "
```

Comment:
The german translation doesn't show, I think it's because of this "fuzzy" thing. Try to open it in PoEdit and uncheck the "needs work" checkbox.